### PR TITLE
Status lifetime

### PR DESCRIPTION
**Pull request by cHeeSaW**
show lifetime of the status in modal-dialog

### CODE REVIEW

File: src/Factory/MonitoringDataDtoFactoryService.php [hunk 1]
```diff
@@ -5,11 +5,28 @@
...
+            $monitoringData->getIdleTimeoutInSeconds(),
+            $monitoringData->getDate(),
+            $monitoringData->getPath(),
+            $monitoringData->getTileExpansionIntervalCount(),
+            $monitoringData->getTileExpansionGrowthExpression(),
+            $monitoringData->getStatusChangedAt()
+        );
+    }
+
+    public function createOutgoingFromIncoming(IncomingMonitoringDataDto $monitoringData): MonitoringDataDto
```
Comment (cawolf) [inline src/Factory/MonitoringDataDtoFactoryService.php]: looks unused to me

**Review by cawolf — CHANGES_REQUESTED** [review]

File: src/Factory/MonitoringDataDtoFactoryService.php (hunk 1 above)
Comment (cHeeSaW) [reply src/Factory/MonitoringDataDtoFactoryService.php]: removed

File: src/Factory/MonitoringDataDtoFactory.php [hunk 2]
```diff
@@ -5,9 +5,12 @@
...
-use App\Dto\MonitoringData as MonitoringDataDto;
+use App\Dto\MonitoringData as IncomingMonitoringDataDto;
+use App\Dto\Outgoing\MonitoringData as OutgoingMonitoringDataDto;
 
 interface MonitoringDataDtoFactory
 {
-    public function create(MonitoringDataDocument $monitoringData): MonitoringDataDto;
+    public function createOutgoingFromDocument(MonitoringDataDocument $monitoringData): OutgoingMonitoringDataDto;
+
+    public function createOutgoingFromIncoming(IncomingMonitoringDataDto $monitoringData): OutgoingMonitoringDataDto;
```
Comment (cawolf) [inline src/Factory/MonitoringDataDtoFactory.php]: the interface still contains the deleted function

**Review by cawolf — CHANGES_REQUESTED** [review]
builds are failing right now

**Review by cawolf — APPROVED** [review]

**Pull request merged**